service/service_construction.go

```diff
@@ -140,8 +145,13 @@ func (s ConstructionService) ConstructionHash(
 		return nil, wrapError(errInvalidInput, "signed transaction value is not provided")
 	}
 
+	wrappedTx := new(signedTransactionWrapper)
+	if err := json.Unmarshal([]byte(req.SignedTransaction), wrappedTx); err != nil {
```

Comment:
```suggestion
	var wrappedTx signedTransactionWrapper
	if err := json.Unmarshal([]byte(req.SignedTransaction), &wrappedTx); err != nil {
```

Reply:
There's other places in the PR where we use `new` when we should be using `var` like this.

Reply:
What's the difference?

Reply:
It's purely stylistic, we use `var` in the codebase so we should be consistent

Reply:
updated